WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -1,15 +1,106 @@
 import Alamofire
 
-@objc public enum WordPressComOAuthError: Int {
-    case unknown
-    case invalidClient
-    case unsupportedGrantType
-    case invalidRequest
-    case needsMultifactorCode
-    case invalidOneTimePassword
-    case socialLoginExistingUserUnconnected
-    case invalidTwoStepCode
-    case unknownUser
+/// Errors returned by `WordPressComOAuthClient`.
+public enum WordPressComOAuthError: Error {
```

Comment:
I think this name is fine and also backwards compatible, but the comment made me wonder whether `WordPressComOAuthClientError` would be better as a name? Or even as a nested type `WordPressComOAuthClient.Error`?

Reply:
I prefer its current name without namespace. It helps Swift compiler to distinguish it with `Swift.Error` and potentially other `Error` types under other namespaces if we are giving all error types namespaces.